docs/source/index.md

```diff
@@ -8,12 +8,51 @@
 :class: only-light
 :::
 
-# PyMC-Marketing - Open Source Marketing Analytics Solution
+<h1 style="text-align: center;">Open Source Marketing Analytics Solution</h1>
 
-**Unlock the power of marketing analytics with PyMC-Marketing – the python based open source solution for smarter decision-making.** Marketing mix modeling and customer lifetime value modules allow businesses to make data-driven decisions about their marketing campaigns. Optimize your marketing strategy and unlock the full potential of your customer data.
+:::{div} sd-text-center
+[![Downloads](https://static.pepy.tech/badge/pymc-marketing)](https://pepy.tech/project/pymc-marketing)
+[![Downloads](https://static.pepy.tech/badge/pymc-marketing/month)](https://pepy.tech/project/pymc-marketing)
+[![Downloads](https://static.pepy.tech/badge/pymc-marketing/week)](https://pepy.tech/project/pymc-marketing)
+:::
+
+<h1 style="text-align: center;">Powered by</h1>
+
+:::{image} _static/labs-logo-dark.png
+:align: center
+:target: https://www.pymc-labs.io
+:scale: 20 %
+:alt: PyMC Labs logo
+:class: only-dark
+:::
+
+:::{image} _static/labs-logo-light.png
+:align: center
+:target: https://www.pymc-labs.io
+:scale: 20 %
+:alt: PyMC Labs logo
+:class: only-light
+:::
+
+---
+
+## 📞 Schedule a Free Consultation for MMM & CLV Strategy
+
+Maximize your marketing ROI with a [free 30-minute strategy session](https://calendly.com/niall-oulton) with our PyMC-Marketing experts. Learn how Bayesian Marketing Mix Modeling and Customer Lifetime Value analytics can boost your organization by making smarter, data-driven decisions.
+
+For businesses looking to integrate PyMC-Marketing into their operational framework, [PyMC Labs](https://www.pymc-labs.com) offers expert consulting and training. Our team is proficient in state-of-the-art Bayesian modeling techniques, with a focus on Marketing Mix Models (MMMs) and Customer Lifetime Value (CLV). Explore these topics further by watching our video on [Bayesian Marketing Mix Models: State of the Art](https://www.youtube.com/watch?v=xVx91prC81g).
+
+We provide the following professional services:
+
+- **Custom Models**: We tailor niche marketing anayltics models to fit your organization's unique needs.
+- **Build Within PyMC-Marketing**: Our team are experts leveraging the capabilities of PyMC-Marketing to create robust marketing models for precise insights.
```

Comment:
I would remove this bullet.

Reply:
Removed in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/bdab3a27c03aec92c4a6b090806ed8b607ffcaa8